Main.cpp

```diff
@@ -1877,8 +1875,11 @@ int main(int argc, char *argv[]) {
     new LuaMan();
     new SettingsMan();
     new TimerMan();
+	new PerformanceMan();
```

Comment:
I'm not entirely sure it matters. 
Thinking of pulling these to the end of `FrameMan::Create`.

Reply:
If they can stay in main I guess I'd rather they do, to make it easy to find where they're initialized and all that. Hiding them in FrameMan may seem nice, but it can make it hard to figure out program initialization flow later.
As far as ordering goes, I was just curious why you ordered them that way. I don't really care as long as it seems reasonable and works.